### PR TITLE
ui

### PR DESCRIPTION
**Pull request by jmcroduac**
Changed design of fragment_camera and fragment_preview_image buttons

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -66,13 +66,15 @@ dependencies {
...
-    implementation 'androidx.palette:palette:1.0.0'
+    implementation 'androidx.palette:palette-ktx:1.0.0'
+
 
     // Navigation library
     def nav_version = "2.3.3"
     implementation "androidx.navigation:navigation-fragment-ktx:$nav_version"
     implementation "androidx.navigation:navigation-ui-ktx:$nav_version"
     implementation "androidx.fragment:fragment-ktx:1.3.0-rc02"
+    implementation "androidx.fragment:fragment:1.3.0-alpha08"
```
Comment (Mino-dev) [inline app/build.gradle:77]: Akala ko di na need to once you include yung fragments for kotlin?

File: app/src/main/java/me/togaparty/notable_opencv/fragments/DashboardFragment.kt [hunk 2]
```diff
@@ -43,10 +43,38 @@ class DashboardFragment : Fragment(), View.OnClickListener {
...
+            R.id.camera_cardview ->
+                return if(ContextCompat.checkSelfPermission(requireContext(),
+                          Manifest.permission.READ_EXTERNAL_STORAGE) ==
+                          PackageManager.PERMISSION_GRANTED){
+                          navController.navigate(
+                                DashboardFragmentDirections.actionDashboardFragmentToCameraFragment())
+                    } else {
+                        setFragmentResult("requestKey",
+                                bundleOf("actionDirection"
+                                        to R.id.action_dashboardFragment_to_cameraFragment.toString()))
```
Comment (Mino-dev) [inline app/src/main/java/me/togaparty/notable_opencv/fragments/DashboardFragment.kt]: Ah. May bagong changes ako dito sa upstream siguro paresolve muna

**Review by Mino-dev — CHANGES_REQUESTED** [review]

File: app/src/main/java/me/togaparty/notable_opencv/fragments/DashboardFragment.kt (hunk 2 above)
Comment (Mino-dev) [reply app/src/main/java/me/togaparty/notable_opencv/fragments/DashboardFragment.kt]: https://github.com/Mino-dev/Notable-OpenCV/blob/b3a09cf064c0c7fc31fd15b4702b9eed6515e56e/app/src/main/java/me/togaparty/notable_opencv/fragments/DashboardFragment.kt#L46

File: app/build.gradle (hunk 1 above)
Comment (jmcroduac) [reply app/build.gradle:77]: Di ko rin alam eh pero may lumalabas kasi na unresolved references dun sa yalantis so dinagdag ko yang line na yan para ma-run ko siya.

**Review by Mino-dev — APPROVED** [review]

**Pull request merged**